packages/contracts-bedrock/test/L2/ExecutingMessageEmitted.t.sol

```diff
@@ -16,7 +16,7 @@ import { ISuperchainTokenBridge } from "interfaces/L2/ISuperchainTokenBridge.sol
 /// @notice Integration test that checks that the `ExecutingMessage` event is emitted on crosschain mints.
 contract ExecutingMessageEmittedTest is CommonTest {
     bytes32 internal constant SENT_MESSAGE_EVENT_SELECTOR =
```

Comment:
Can we use the selector property on the event itself rather than hardcoding it and needing it to change?